crates/storage/db/src/implementation/mdbx/mod.rs

```diff
@@ -61,9 +63,11 @@ impl DatabaseEnvKind {
 
 /// Arguments for database initialization.
 #[derive(Clone, Debug, Default)]
-pub struct DatabaseArguments {
+pub struct DatabaseArguments<R: RangeBounds<usize> + Clone = Range<usize>> {
```

Comment:
this generic is a bit weird, imo we can just use `Range<usize>` without the generic

Reply:
fixed, thanks